style.css

```diff
@@ -90,4 +91,22 @@ margin: 0;
 }
 .flex-container img{
 width: 45%;
+}
+.container-footer{
+text-align: center;
+background-color:#ece7e7;
+}
+.container-footer h5{
+color:black;
+font-weight: bold;
+}
+.logo-wrap{
```

Comment:
Why do you have styling for class, that isn't used anywhere in index files code?

Reply:
Looks, like I created the class and renamed it or deleted it. But made changes regarding class naming based on BEM suggestion.